jenkins/Makefile

```diff
@@ -3,7 +3,13 @@
 
 BRANCH := $(shell cat ./branch.name)
```

Comment:
Two different files use it, `jenkins/Makefile` and `jenkins/run`.  So having a file seems reasonable, and ... it allowed me to make an isolated change when testing to ensure the PR does not disrupt `main`.

Reply:
Apparently, there's a _third_ usage:  `server/pbenchinacan/container-build.sh`  😀

In any case, I appreciate what this mechanism enables; the part that is unreasonable is that it forces you to mark the PR as "DO NOT MERGE".  I don't really want to be reviewing a submission which cannot be "accepted".  And, it's annoying to know that, once the PR gets all of its approvals, the branch will have to be modified (and probably re-approved) before it can be merged.  So, I'm wondering if there isn't an alternative approach.